packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -96,12 +104,21 @@ export default defineComponent({
     const store = useStore()
     const { selectedResourcesIds } = useSelectedResources({ store })
 
-    const spaces = computed(
-      () =>
-        store.getters['runtime/spaces/spaces']
-          .filter((s) => isProjectSpaceResource(s))
-          .sort((a, b) => a.name.localeCompare(b.name)) || []
+    const runtimeSpaces = computed(
+      () => store.getters['runtime/spaces/spaces'].filter((s) => isProjectSpaceResource(s)) || []
     )
+
+    const sortFields = [availableSortFields[0], availableSortFields[1]]
```

Comment:
Would love to see the other sort options for project spaces as well... for size we could utilize the used quota. And the `mdate` actually has a value present... not sure how it's updated though.

Reply:
(fine as a followup if you want to, but in that case please create an issue)

Reply:
the `size` prop for `SpaceResource`s is empty... so you could just write the `spaceQuota?.used` into the `size` prop in `buildSpaceResource`. That should make the size work.

Reply:
I've created https://github.com/owncloud/web/issues/8439. `mdate` would also be nice, but needs some adjustments as well.